.github/workflows/ghcr-build.yml

```
@@ -35,24 +35,33 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v4
 
+      - name: Set up QEMU for cross-platform builds
+        uses: docker/setup-qemu-action@v2
```

```suggestion
        uses: docker/setup-qemu-action@v3
```
Why not use the current version